docs/tools/integrations/integrations.md

```diff
@@ -0,0 +1,88 @@
+---
+id: integrations
+title: 🔮 Integrations
+sidebar_position: 2
+---
+
+# 🔮 Integrations
+
+A list of services and projects that integrated the LUKSO network or deliver ecosystem tools.
+
+## RPC Providers
+
+<table><thead>
```

Comment:
I think for these tables, we can have them as pure markdown. Since we are not writing complex layout and text inside the <td>.

It will be easier to maintain